src/ui/GenioWindow.cpp

```diff
@@ -3521,6 +3521,12 @@ GenioWindow::_ShowSelectedItemInTracker()
 			status = parentDirectory.GetRef(&ref);
 			if (status == B_OK)
 				status = _ShowInTracker(ref);
+			if (status == B_OK) {
+				entry_ref itemRef;
+				itemEntry.GetRef(&itemRef);
```

Comment:
Be aware that we are near to merge a PR and this part may need to be adapted (but should be straightforward)